client/messaging/message-layout.tsx

```diff
@@ -41,11 +42,13 @@ interface MessageTimestampProps {
 }
 
 export const MessageTimestamp = (props: MessageTimestampProps) => (
-  <Timestamp title={longTimestamp.format(props.time)}>
-    <Separator>[</Separator>
-    {shortTimestamp.format(props.time)}
-    <Separator>] </Separator>
-  </Timestamp>
+  <Tooltip text={longTimestamp.format(props.time)} position='top'>
+    <Timestamp>
```

Comment:
I feel like whenever we use this thing, we should probably also be including the tooltip text in an aria attribute (e.g. `aria-label`). Tooltips aren't particular accessible to screenreaders

Reply:
(With a wrapper element you could probably do this as part of the Tooltip component actually, although you may want to make it customizable for things that have more content than just what is in the tooltip. Or maybe there's a better ARIA attribute, I dunno)